### ISSUE TITLE
Broken since 2024.03.12.0.1

### ISSUE THREAD

**Issue by cyberrbob** [bazelbuild/intellij#6384]
### Description of the bug:

Update from 2024.02.27.0.1 to 2024.03.12.0.1 broke it. Project opens but I see new warning like "Warning:Issues collecting info from C++ compiler" and it can not resolve references/links in the file (so highlight lots of code with red cursive), so basically all the navigation within project is broken. Manually downgraded back to 2024.02.27.0.1 and it works again.

@enovozhilov @ujohnny I was told to mention you here about this issue

### Which category does this issue belong to?

CLion

### Which Intellij IDE are you using? Please provide the specific version.

CLion 2023.2.3

### What programming languages and tools are you using? Please provide specific versions.

c++17

### What Bazel plugin version are you using?

2024.02.27.0.1

**Comment by ujohnny** [comment]
@cyberrbob thank you for reaching out to us here, could you please invoke "Show compiler info" action on the file with red code, do you have many `-ibazel` there? Another option -- could you please enable "Resharper" engine in Advanced settings and see if there's no red code?

**Comment by SimplyTheOther** [comment]
Adding that key and resyncing worked, thanks.

**Comment by SimplyTheOther** [comment]
Hi, after updating to the latest Bazel plugin version (which I assume is the "better workaround" you mentioned), disabling via that key alone no longer works.

I can only successfully get the files analysed if I explicitly disable both the original virtual includes handling *and* the new virtual includes handling one.

**Comment by cyberrbob** [comment]
Hi, just wanted to confirm that @SimplyTheOther mentioned, only adding these two lines:
```
bazel.sync.resolve.virtual.includes=false
bazel.sync.collect.virtual.includes.hints=false
```
as custom properties fixes the problem. When can we wait a real fix ? Can you please post a ticket number or something for tracking.

**Comment by ujohnny** [comment]
We disabled these changes by default with https://github.com/bazelbuild/intellij/pull/6534. If you still experience this issue, please reopen this ticket.

Edit: The fix is available in [v2024.07.02-RC-1](https://github.com/bazelbuild/intellij/releases/tag/v2024.07.02-rc-1)
